src/main/java/businessObjects/GraphQlQueries.java

```diff
@@ -24,5 +24,5 @@
 package businessObjects;
 
 public final class GraphQlQueries {
-    public static final String GHSA_SECURITY_ADVISORY_QUERY = "query { securityAdvisory(ghsaId: \"%s\") { ghsaId summary cwes(first : 1) { nodes { cweId } } } }";
+    public static final String GHSA_SECURITY_ADVISORY_QUERY = "query { securityAdvisory(ghsaId: \"%s\") { ghsaId summary identifiers{ type value } cwes(first : 1) { nodes { cweId } } } }";
```

Comment:
The SBOM pique model relies on the exact original query. Could you revert this method to the original query and add another String representing your new query?